rhc.gemspec

```diff
@@ -30,7 +30,11 @@ Gem::Specification.new do |s|
     sep
   ].join("\n")
 
-  s.add_dependency              'net-ssh',      '>= 2.0.11', '<= 2.9.2'
+  if RUBY_VERSION <= "1.9.3"
```

Comment:
I don't think you can do this, since this will get evaluated just during ```gem build``` and will be fixed in the metadata.